changelog/6082.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix line detection for properties in doctest tests
```

Comment:
I suggest we link to the issue:

```rst
Fix line detection for doctest samples inside ``property`` docstrings, as workaround to `bpo-17446 <https://bugs.python.org/issue17446>`__.
```

Also good news is that there's a PR open: https://github.com/python/cpython/pull/3419

Finally, could you please rebase your changes and squash all commits? Let me know if you rather us do it. 👍

Reply:
@nicoddemus Thanks! I've added a link to the issue as you suggested and squashed all commits. I saw this PR to cpython, however, it's not active since Jan 2, 2018 :(

Reply:
Sorry about that -- I just haven't been able to figure out what the reviewer wanted in it.  Will try to get it in again soon.